dapps/react-dapp-v2/src/components/app/index.tsx

```diff
@@ -53,6 +53,14 @@ export const SToggleContainer = styled.div`
   }
 `;
 
+export const SDropDownContainer = styled.div`
+  paddingtop: 20;
```

Comment:
Do these props work like this with styled-components? Seems like they're missing hyphens
```suggestion
  padding-top: 20;
```